### ISSUE TITLE
This missing a lot of features! 

### ISSUE THREAD

**Issue by minanagehsalalma** [xtr4nge/module_karma#1]

**Comment by H4xl0r** [comment]
Go home pls

**Comment by minanagehsalalma** [comment]
> Go home pls

If i only knew the way  ;(

**Comment by H4xl0r** [comment]
:-) 
list of features missing for you ?

**Comment by minanagehsalalma** [comment]
> :-)
> list of features missing for you ?

I won't  tell you  after that rude reply of you !

**Comment by H4xl0r** [comment]
what a loss

**Comment by minanagehsalalma** [comment]
> what a loss

yeah good luck next time and always remember being kind to others doesn't cost anything.

**Comment by H4xl0r** [comment]
.-) module Karma Activated

**Comment by minanagehsalalma** [comment]
> .-) module Karma Activated

Haha so wait for your turn.
As the most of the time i am kind.

**Comment by H4xl0r** [comment]
:-) so we go on with the game ok :-) 
tell me what is missing for you, maybe we can make some module :-)
the mana module is way better anyway

**Comment by minanagehsalalma** [comment]
here you go 
https://github.com/hak5/wifipineapple-modules/issues/85
https://github.com/hak5/wifipineapple-modules/issues/88

**Comment by H4xl0r** [comment]
Ok the first Request

you could do this with the portal , there is an recon script , its capturing all the requested infos ....
into an json db

i will pull the rel i have overworked from the captive portal, so it will pop up auto on all devices

Szenario simple and clever:
use mana with captive 
user get authenticated in his pocet without knowledge 
the page pops up you got your info ....
(using this to serve drivebys ... no one can stop that auto pop )

**Comment by H4xl0r** [comment]
The second request is interessting , so you wanna have the wpa handshakes of the devices , that are not answering to open ap requests.... this would be possible without big hussle ,
but it wont help you further .... you still have to crack those handshakes.

better way would be the evil twin, but im still interessted in the idea 
nice share

this could be done with an second mana module running for the scured ones , possible,
but for good work you would need 2cards for station and 1 for monitor 

Client calls wpa wifi ----> Manna says yes abd starts airodump on the ssid and channel --->  Client connects and we got the handshake, verify handshake with airodump (is captured?) if yes drop connection set to blacklist send wpa2 handshake to online cracker (optional).... next 


im bussy at the moment but i will take a look on it would be interessting 
so i would be able to capture the handshakes crack them in the cloud and ask the people around me 
hey is your wifi dadada with pw nanana
idiot 

There is allready a function for that in manna , so its already possible

https://github.com/sensepost/hostapd-mana/blob/master/hostapd/hostapd.conf#L55
https://github.com/sensepost/hostapd-mana/blob/master/hostapd/hostapd.conf#L1204

do you get it own your own ? 
simply change those settings and it should work as wished ? :-) 
anyway i will do an wpa module next free time 
want to use it also

**Comment by minanagehsalalma** [comment]
"2cards for station and 1 for monitor"

Nob not really this won't need more than one card.. the most of cards are capable of brodcasting an ap and sniffing at the same time .
You can do that using 
https://youtu.be/47q3JmtxWh4?t=1m52s
But without the karma probs part.

Also if you liked these then sure you will like this 
https://github.com/hak5/wifipineapple-modules/issues/89

**Comment by H4xl0r** [comment]
> 
> 
> "2cards for station and 1 for monitor"
> 
> Nob not really this won't need more than one card.. the most of cards are capable of brodcasting an ap and sniffing at the same time .
> You can do that using
> https://youtu.be/47q3JmtxWh4?t=1m52s
> But without the karma probs part.
> 
> Also if you liked these then sure you will like this
> [hak5/wifipineapple-modules#89](https://github.com/hak5/wifipineapple-modules/issues/89)

he is using some kind of 
https://github.com/0x90/wifi-arsenal/blob/master/handshakeharvest-K1-K2-K2016-4-0.sh
also a nice script by the way

**Comment by H4xl0r** [comment]
the other one is way unneded in my opinoin 
but also an idea ....

the monitor mode and ap with 2 wifi adapters was ment to be used with mana normal for nornal ssid attack and the other for the mana wpa2 ap and the handshakes ...
so you would be able to capture everything
the other wont work with wpa and non wpa as one ap calling for all asked ssids ...
